src/features/bivariate_manager/components/ConnectedBivariateMatrix/ConnectedBivariateMatrix.tsx

```diff
@@ -125,7 +127,7 @@ const ConnectedBivariateMatrix = () => {
         debouncedShowTooltip({ x: e.clientX, y: e.clientY });
       }
     },
-    [xGroups, yGroups, setMatrixSelection, debouncedShowTooltip, selectedCell],
```

Comment:
Did you mean it?
i'm asking since `selectedCellRef` is still in the callback

Reply:
yep, as selectedCellRef is ref, that is not needed to be synced